kube/services/jobs/google-manage-account-access-cronjob.yaml

```diff
@@ -0,0 +1,85 @@
+spec:
+  schedule: "@hourly"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    spec:
+      # not yet supported - backOffLimit: 3
+      template:
+        spec:
+          serviceAccountName: useryaml-job
+          volumes:
+            - name: creds-volume
+              secret:
+                secretName: "fence-creds"
+            - name: config-volume
+              secret:
+                secretName: "fence-secret"
+            - name: json-secret-volume
+              secret:
+                secretName: "fence-json-secret"
+            - name: fence-google-app-creds-secret-volume
+              secret:
+                secretName: "fence-google-app-creds-secret"
+            - name: fence-google-storage-creds-secret-volume
+              secret:
+                secretName: "fence-google-storage-creds-secret"
+            - name: fence-jwt-keys
+              secret:
+                secretName: "fence-jwt-keys"
+            - name: fence-yaml
+              configMap:
+                name: fence
+            - name: config-helper
+              configMap:
+                name: config-helper
+          containers:
+          - name: fence
+            image: quay.io/cdis/fence:master
+            imagePullPolicy: Always
+            env:
+              - name: PYTHONPATH
+                value: /var/www/fence
+            volumeMounts:
+              - name: "creds-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/creds.json"
+                subPath: creds.json
+              - name: "config-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/local_settings.py"
+                subPath: local_settings.py
+              - name: "json-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_credentials.json"
+                subPath: fence_credentials.json
+              - name: "fence-google-app-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_app_creds_secret.json"
+                subPath: fence_google_app_creds_secret.json
+              - name: "fence-google-storage-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_storage_creds_secret.json"
+                subPath: fence_google_storage_creds_secret.json
+              - name: "fence-yaml"
+                mountPath: "/var/www/fence/user.yaml"
+                subPath: user.yaml
+              - name: "fence-jwt-keys"
+                readOnly: true
+                mountPath: "/fence/keys"
+              - name: "config-helper"
+                readOnly: true
+                mountPath: "/var/www/fence/config_helper.py"
+                subPath: config_helper.py
+            command: ["/bin/bash"]
+            args:
+              - "-c"
+              - |
+                fence-create google-manage-account-access
```

Comment:
Add another block here:
```
if [[ $? != 0 ]]; then
  echo "WARNING: non zero exit code: $?"
fi
```
So the script always succeeds (always has exit code 0) - just logs failures, and we avoid Kubernetes annoying retry logic kicking in ...